nupic/algorithms/anomaly.py

```diff
@@ -123,25 +126,9 @@ def compute(self, activeColumns, predictedColumns,
                               	(used in anomaly-likelihood)
     @return the computed anomaly score; float 0..1
```

Comment:
The likelihood probability is expressed as a fraction from 0.0 to 1.0, right? Wasn't it doing that before as well?